README.md

```diff
@@ -50,14 +46,13 @@ Required options:
   - `--s3-bucket`: The S3 bucket containing your org's kubectl config file
   - `--s3-key`: The S3 key of your org's kubectl config file
 
-### Initializing a project
+### Setting up an existing project
 
-* `hokusai init` - Writes hokusai project config to `hokusai/config.yml`, creates test, development and production yaml files alongside it, and adds a Dockerfile to the current directory.
+* `hokusai setup` - Writes hokusai project config to `hokusai/config.yml`, creates test, development and production yml files alongside it, and adds a Dockerfile to the current directory.
 
 Required options:
   - `--aws-account-id`: Your AWS account ID - can be found in your AWS account console.
```

Comment:
Should we add the other options - mongodb, etc. so people realize they are here without running `--help`?